code/modules/projectiles/projectile/beams.dm

```diff
@@ -85,6 +85,7 @@
 	icon_state = "omnilaser"
 	hitsound = 'sound/weapons/tap.ogg'
 	nodamage = 1
+	damage = 0
 	damage_type = STAMINA
```

Comment:
Does this still do stamina damage if you specify damage to 0?

Reply:
nodamage will override it all so it does nothing
![image](https://user-images.githubusercontent.com/15887760/94993247-e4adf500-058f-11eb-9959-b342d0c6d25d.png)